setup.py

```diff
@@ -6,7 +6,7 @@
     long_description = fh.read()
 
 dependencies = [
-    "blspy>=0.9",
+    "chia_rs==0.3.3",
```

Comment:
Can we be a bit looser?  Otherwise this has to update in lock step with blockchain.

Reply:
Done.